src/WorkOS.net/Services/UserManagement/Entities/User.cs

```diff
@@ -48,20 +42,8 @@ public class User
         /// <summary>
         /// The timestamp when the user's email was verified. Email verification is only applicable to unmanaged users.
```

Comment:
Can we update this comment as well to reflect `EmailVerified`, as it is no longer a timestamp?

Reply:
updated